finder.js

```diff
@@ -1,9 +1,18 @@
-import config from './config';
+import fs from 'fs';
+import {EOL} from 'os';
 import clone from './lib/clone';
 import search from './lib/search';
 import serviceCatalogue from './lib/serviceCatalogue';
 import isFrontendService from './lib/utils/isFrontendService';
 
+try {
+  var config = require('./config.json');
+}
+catch(err) {
+  process.stdout.write(`No config.json file found ${EOL}`);
```

Comment:
I don't think it's necessary. We're only reporting a missing file and it's not in function or anything. That makes it a UX sugar thing and throwing an error doesn't log to the console nicely.